SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp

```diff
@@ -273,6 +273,17 @@ bool FLocalDeploymentManager::LocalDeploymentPreRunChecks()
 		}
 	}
 
+	if (!IsServiceRunningAndInCorrectDirectory()) {
+		if (FMessageDialog::Open(EAppMsgType::YesNo, LOCTEXT("StopSpatailServiceFromDifferentProject", "Another SpatialOS service is already running in another project. Would you like to stop it?")) == EAppReturnType::Yes)
```

Comment:
@jessicafalk could we confirm this is fine with Xcode?

Reply:
that should be fine ;) no exclamation mark that can cause confusion